plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergColumnHandle.java

```diff
@@ -166,4 +169,28 @@ public String toString()
     {
         return getId() + ":" + getName() + ":" + type.getDisplayName();
     }
+
+    public static IcebergColumnHandle getPathColumnHandle()
```

Comment:
`pathColumnHandle` instead of `getPathColumnHandle`

we are not exposing here an internal field of the `IcebergColumnHandle` POJO.

Reply:
updated @findinpath